cmd/alert.go

```diff
@@ -1,20 +1,45 @@
 package cmd
 
 import (
+    "fmt"
+    "errors"
 	"github.com/anynines/a9s-aha-cli/pkg/alert"
+    "github.com/anynines/a9s-aha-cli/pkg/notify"
 
 	"github.com/spf13/cobra"
 )
 
-func newCmdAlert() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "alert",
-		Short: "Alert in Slack channel if any alerts present",
-		Long:  `Alert in Slack channel if any alerts present.`,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return alert.Stale()
-		},
-	}
+var newCmdAlert = &cobra.Command{
+  Use:   "alert",
+  Short: "Alert in Slack channel if any alerts present",
+  Long:  `Alert in Slack channel if any alerts present.`,
+  RunE: func(cmd *cobra.Command, args []string) error {
+    status, _ := cmd.Flags().GetBool("notify")
+    check(status)
+    return nil
+  },
+}
+
+func init() {
+    RootCmd.AddCommand(newCmdAlert)
+    newCmdAlert.Flags().BoolP("notify", "n", false, "Notify the alert message on Slack.")
+}
 
-	return cmd
+func check(fstatus bool) error {
```

Comment:
cmd/ should be kept free from any logic. Its just the entrypoint for the CLI to trigger some actions, therefore alert.Stale() existed.